returnn/datasets/meta.py

```diff
@@ -1391,6 +1391,7 @@ def __init__(
         seq_tag_delim=";",
         remove_in_between_postfix=None,
         repeat_in_between_last_frame_up_to_multiple_of=None,
+        pad_truncate_data_to_multiple_of_target_len=None,
```

Comment:
I still don't like the name. `truncate` is misleading here. `truncate` usually means sth different (e.g. see [`torch.trunc`](https://pytorch.org/docs/stable/generated/torch.trunc.html)).

In RETURNN, we have the function `slice_pad_zeros` which does what you want here. So we use the term `slice`. Instead of `slice`, you might also want to call this `narrow` ([`torch.narrow`](https://pytorch.org/docs/stable/generated/torch.narrow.html#torch.narrow)).

So maybe:
```suggestion
        pad_narrow_data_to_multiple_of_target_len=None,
```